module/scripts/combat/effects.js

```diff
@@ -0,0 +1,94 @@
+// module\scripts\combat\effects.js
+export { applyBleed, applyFire, applyPoison, applySuffocation };
+
+async function changeHealth(actor, amount, name, remidy = "") {
+    // Ensure `amount` is a number
```

Comment:
There is already a method for heal and damage

Reply:
Yes, but applyDamage needs a location. Poison or suffocation should not be reduced by SP. Wouldnt I then need a function like calculateDamageWithoutLocation?
For Fire I would then probably just add a damageProperties.Fire property that would also reduce the SP no matter if the sp is bigger than the damage. Also I would then need to check all equipped weapons.
I am unsure if this would not make it more difficult than having it seperately.

Reply:
I can adjust it, so that dialogData is optional, because there is no data present that you need